README.md

```diff
@@ -1,114 +1,90 @@
-# Quick documentation
-- [First Steps](#first-steps)
-  * [Querying via bugzilla-cli](#querying-via-bugzilla-cli)
-  * [Bashing](#bashing)
-  * [Pythonic](#pythonic)
-    + [Setting env](#setting-env)
-    + [Query Networking Bugs Script](#query-networking-bugs-script)
-    + [Documentation](#documentation)
-
-This is quick notes page as reference for using `bugzilla-cli` command line.  
-`bugzilla-cli` is based on [python-bugzilla project](https://github.com/python-bugzilla/python-bugzilla) which we will use for some `specific queries` as well.
-
-## First Steps
-First, generate the `api key` to communicate with bugzilla using `python-bugzilla` module.  
-Go to https://bugzilla.redhat.com and click: **Username -> Preferences -> API Keys**.  
-
-**ATENTION:**  
+# Quick reference
+- [Download source code](#download-source-code)
+- [Install bugzilla client](#install-bugzilla-client)
+- [Install jira client](#install-jira-client)
+- [Run network_bugs_overview script](#run-network_bugs_overview-script)
+- [Documentation](#documentation)
+
+This is a quick reference for the `network_bugs_overview` command line.
+`network_bugs_overview` is based on [python-bugzilla project](https://github.com/python-bugzilla/python-bugzilla) and on [jira library for Python](https://github.com/pycontribs/jira).
+
+It fetches open bugs assigned to our team members from our bugzilla and our jira servers, analyzes them and prints out a per-developer summary in the order from the currently least-loaded to currently the most-loaded team member. 
+
+In particular, it queries:
+- bugzilla for existing bugs (the creation new bugs will soon be disabled on our bugzilla server), 
+- jira (OCPBUGS project) for bugs and
+- jira (RHOCPPRIO project) for escalations.
+
+## Download source code
+```
+$ git clone https://github.com/dougsland/bz-query && pushd bz-query
```

Comment:
@ricky-rav I think we need to refine the install a little more.
I tried followoing the steps described here and end up with this error:
```
env ❯ ./network_bugs_overview -h
Traceback (most recent call last):
  File "/home/vagrant/dev/douglas_bugzilla/tmp/bz-query/./network_bugs_overview", line 8, in <module>
    from dateutil import parser
ModuleNotFoundError: No module named 'dateutil'
```

How about we create a file called requirements.txt with the dependencies we need:

```
jira
python-dateutil
```

and the as part of the install add `pip install .`